core/src/main/java/apoc/coll/Coll.java

```diff
@@ -963,13 +964,10 @@ public List<List<Object>> pairWithOffsetFn(@Name("coll") List<Object> values, @N
         return result;
     }
 
-    @Procedure
-    @Description("apoc.coll.pairWithOffset(values, offset) - returns a list of pairs defined by the offset")
+    @Procedure("apoc.coll.pairWithOffset")
+    @Description("Returns a list of pairs defined by the offset.")
     public Stream<ListResult> pairWithOffset(@Name("values") List<Object> values, @Name("offset") long offset) {
```

Comment:
I noticed that the first parameter is named `coll` in the 3 different places it is listed in docs, but `values` here, and in the signature and input parameters on the docs page. Same applies to `apoc.coll.partition` above.

Not sure which way we would like to change it, but should be consistent.